lib/utils.py

```diff
@@ -92,3 +92,40 @@ def inline_anova(anova,
 		if pythontex_safe:
 			inline = inline.replace("\\","\\\\")
 	return inline
+
+def bold_max_row_value(df, df_tex: str):
+    """
+    Wraps the maximum value for each row with \textbf{} in a df.to_latex() string.
+
+    Parameters
+    ----------
+    df :  pandas.DataFrame
+            Pandas DataFrame object containing at least one column with numerical values.
+
+    df_tex: string
+                The output of df.to_latex(). The values if this string will be changed such that the maximum values of
```

Comment:
“String as returned by `df.to_latex()`”. I also think it should be “The values IN” rather than “if”. Also maybe make the explanation longer, i.e. what happens to string or datetime columns, do they get fully skipped by your iterator? Maybe also test it so that if it fails it fails nicely.